README.md

```diff
@@ -157,6 +157,19 @@ You can find a full description of errors including exit codes here: <https://do
 Yes!
 Extra pre-built packages with a `.full` suffix are available and contain split debuginfo, documentation files, and extra binaries like the `detect-wasi` utility.
 
+## Telemetry collection
+
+Some crate installation strategies may collect anonymized usage statistics by default.
+Currently, only the name of the crate to be installed, its version, and the target platform triple are sent to endpoints under the `https://warehouse-clerk-tmp.vercel.app/api/crate` URL when the `quickinstall` artifact host is used.
```

Comment:
Yeah, this url makes me really sad, but I think it is correct.

I should really finish that work to switch onto fly.io+influxdb at some point (or use cloud flare/whatever). At the moment my head is deep in python/js/vr/robotics labs though, so realistically it's not going to happen unless someone steps up to do it.